libs/langchain/langchain/agents/json_chat/base.py

```diff
@@ -15,7 +15,7 @@ def create_json_chat_agent(
     llm: BaseLanguageModel,
     tools: Sequence[BaseTool],
     prompt: ChatPromptTemplate,
-    stop_sequence: bool = True,
+    stop_sequence: Union[bool, List[str]] = True,
```

Comment:
Could you confirm you're OK with this change?

I noticed that there was an existing stop_sequence in this API. If so, I thin we should make the other APIs consistent

Reply:
LGTM， better than another stop parameter